satellite/peer.go

```diff
@@ -153,6 +153,9 @@ type Peer struct {
 		Database  storage.KeyValueStore // TODO: move into pointerDB
 		Service   *metainfo.Service
 		Endpoint2 *metainfo.Endpoint
+
+		// TODO(michal) most probably not best place
+		ProjectUsage *accounting.ProjectUsage
```

Comment:
what do you mean by this comment? that the `accounting.ProjectUsage` code should be moved into the metainfo package?

Reply:
I had a dilemma how to organize this only in `peer.go` but `Accounting` should be ok. Comment removed.